docs/index.bs

```diff
@@ -3145,6 +3146,22 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                   1. Let |response| be |requestResponse|'s response.
                   1. If |client| is not null, |response|'s [=response/type=] is "`opaque`", and [=cross-origin resource policy check=] with |request|'s [=request/origin=], |client|, "", and |response|'s [=filtered response/internal response=] returns <b>blocked</b>, then return null.
                   1. Return |response|.
+          1. Else if |source| is {{RouterSource/"race-network-and-fetch-handler"}}, and |request|'s [=request/method=] is \`<code>GET</code>\` then:
+
+              Note: In order to avoid duplicated fetches to |request|, the user agent may reuse |raceNetworkRequestResponse| as the result of |e|'s [=FetchEvent/potential response=], or the fallback request.
```

Comment:
nit but |e| might not be defined yet.  I guess it fair to say `FetchEvent` here.